src/lib/components/documents/DocumentListItem.svelte

```diff
@@ -104,15 +133,36 @@ It's deliberately minimal and can be wrapped in other components to add addition
   }
 
   .thumbnail {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    position: relative;
+  }
+
+  .thumbnail img,
+  .thumbnail .fallback {
     border-radius: 0.125rem;
     border: 1px solid var(--gray-2, #cbcbcb);
     background: var(--white, #fff);
     box-shadow: 0px 2px 4px 0px rgba(0, 0, 0, 0.1);
   }
 
+  .thumbnail p {
+    color: var(--gray-5);
+    font-size: var(--font-xs);
+  }
+
+  .thumbnail .tabs {
+    position: absolute;
+    left: -0.5rem;
+    top: 0.5rem;
+    display: flex;
+    flex-direction: column;
+    gap: 0.125rem;
+  }
+
```

Comment:
I simplified tab placement by absolutely positioning a container element. Positioning each tab required assuming its height as `0.5rem`; this approach responds to any potential changes in tab height.